src/store/actions/api.actions.js

```diff
@@ -1,6 +1,8 @@
 import client from '../api';
 import apiSchema from '../../configurations/apiSchema'; // @TODO: REMOVE
 import {
+  DELETE_API_REQUEST,
```

Comment:
we use ACTION_NAME_START, ACTION_NAME_SUCCESS, ACTION_NAME_FAILURE naming convention for async actions. I believe it makes sense to use it here too